chain/indexer/src/streamer/mod.rs

```diff
@@ -296,8 +341,11 @@ pub(crate) async fn start(
             .path()
             .join("indexer");
 
-    // TODO: implement proper error handling
-    let db = DB::open_default(indexer_db_path).unwrap();
+    let db = match DB::open_default(indexer_db_path) {
+        Ok(db) => db,
+        Err(err) => panic!("Unable to open indexer db: {:?}", err),
+    };
```

Comment:
```suggestion
    let db = DB::open_default(indexer_db_path).expect("Unable to open indexer db");
```

Reply:
Will it include the original error message? I wanted it to be shown.

Reply:
Yes, it will. That is the whole point of `.expect`